FWCore/Framework/interface/ModuleTypeResolverMakerFactory.h

```diff
@@ -0,0 +1,15 @@
+#ifndef FWCore_Framework_itnerface_ModuleTypeResolverFactory_h
+#define FWCore_Framework_itnerface_ModuleTypeResolverFactory_h
+
+#include "FWCore/PluginManager/interface/PluginFactory.h"
+
+#include <string>
+#include <vector>
+
+namespace edm {
+  class ModuleTypeResolverMaker;
+
+  using ModuleTypeResolverMakerFactory = edmplugin::PluginFactory<ModuleTypeResolverMaker*()>;
```

Comment:
Could this be `ModuleTypeResolverMaker const*`

Reply:
Seems to work, thanks.